client/css/style.css

```diff
@@ -36,21 +36,27 @@ section {
     background: #eee;
 }
 #scoreboard{
+<<<<<<< HEAD:client/assets/css/style.css
+    background-color:rgba(20,20,20,0.3);
+=======
     background-color:rgba(20,20,20,0.9);
+>>>>>>> 97100e290d41fb761e0897dd3183e08a64dd1fb1:client/css/style.css
     display:flex;
     height:400px;
     width:250px;
     flex-direction:column;
 }
 #player-list{
     list-style-type: none;
-    width:100%;
+    background-color: white;
+    width: 160px;
+    text-transform: uppercase;
```

Comment:
People may want to choose some uppercase, some lowercase characters, don't force the visual here.